tests/kernel/sched/preempt/src/main.c

```diff
@@ -49,7 +49,7 @@ const enum { METAIRQ, COOP, PREEMPTIBLE } worker_priorities[] = {
 #define NUM_THREADS ARRAY_SIZE(worker_priorities)
 
 #ifdef CONFIG_COVERAGE
-#define STACK_SIZE (512 + CONFIG_TEST_EXTRA_STACKSIZE)
+#define STACK_SIZE 4096
```

Comment:
This shouldn't need to change?

Reply:
In this test’s prj.conf , it defines a small CONFIG_TEST_EXTRA_STACKSIZE.